dashboard/app/views/levels/_external.html.haml

```diff
@@ -32,8 +33,9 @@
 - if video && use_large_video_player
   .clear
   .video-container.external-video-container{{style: 'max-width: 970px; margin: auto'}}
-  .button-box{{style: 'width: 970px; margin: auto'}}
-    %a.btn.btn-large.btn-primary.next-lesson.submitButton.pull-right= t('continue')
+    .buttons.below-video-right{style: "float: right; max-width: 20%"}
```

Comment:
Similarly here, is the `.buttons.below-video` what's replacing the `.button-box` class?

Reply:
Thanks so much for your question!
Actually, I was not looking at an `external` type file with `use_large_video_player` set to `true`.
Once I viewed a page that used this logic, I saw that `button-box` is still needed!

I'm going to add `pull-right` and also switch the order of the 'Download' and 'Continue' buttons so when tabbing, the user will navigate from left to right.